src/style.css

```
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 900;
+}
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.form-search {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-icon {
+  border: none;
+  display: block;
+  position: absolute;
+  background-color: transparent;
+}
+
+.search-icon-big {
+  left: calc(62px / 2);
+  top: 50%;
+  transform: translateX(-50%) translateY(-50%);
+}
+
+.search-icon-small {
+  left: calc(34px / 2);
+  top: 50%;
+  transform: translateX(-50%) translateY(-50%);
+}
+
+.small-img {
+  width: 11px;
+  height: 11px;
+}
+
+.search-input {
+  display: block;
+  width: 100%;
+  outline: none;
+  font-family: inherit;
+  font-weight: 900;
+  color: rgb(61, 78, 97);
+  border: 1px solid rgb(225, 231, 237);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background-repeat: no-repeat;
+}
+
+.big_input_search {
+  height: 70px;
+  line-height: 22px;
+  font-size: 16px;
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 26px 25px;
+  padding: 25px 0 26px 62px;
+  margin: 20px 0;
+}
+
+.big_input_search::placeholder {
+  text-shadow: none;
+  color: rgb(61, 78, 97);
+  font-size: 16px;
+  font-family: inherit;
+  font-weight: 300;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.small_input_search::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  color: rgb(61, 78, 97);
+  text-shadow: none;
+}
```

Try to combine common styles for both placeholders
```suggestion

.big_input_search::placeholder,
.small_input_search::placeholder {
  font-family: inherit;
  font-weight: 300;
  color: rgb(61, 78, 97);
  text-shadow: none;
}
```